components/profileName/profileName.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { View, Text, StyleSheet, ScrollView } from 'react-native';
+
+const ProfileName = () => {
+    return (
+        <ScrollView style={styles.ScrollView}>
+            <View style={styles.body}>
+                <Text style={styles.titleText}>Максим Чернявский</Text>
+            </View>
+        </ScrollView>
+    );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Developers use different patterns in development. I like when we split components code and styles code.
You should move styles to ProfileName.styles.js file and import it here, the same should work for all components too. (You need to fix AboutMe files too)